### PR TITLE
Places Story is ready for ah pull request!

### PR DESCRIPTION
**Pull request by amcjen**

### CODE REVIEW

File: Apps/MergedContacts [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 025d114952c195e56f9b0c46a9d1aa1d4d4a789f
+Subproject commit 0931152e00c6cfc82a956ca8ca22d1dcac2abc79
```
Comment (ctide) [inline Apps/MergedContacts:1]: Is this intentional?

File: Collections/Places/places.js [hunk 2]
```diff
@@ -7,89 +7,109 @@
 *
 */
 
-// merge places from connectors
+// merge contacts from connectors
```
Comment (ctide) [inline Collections/Places/places.js]: typo

File: Apps/MergedContacts (hunk 1 above)
Comment (amcjen) [inline Apps/MergedContacts:1]: Nope, thx.

File: Collections/Places/places.js (hunk 2 above)
Comment (amcjen) [inline Collections/Places/places.js]: fixed

**Pull request merged**